src/plugins/support.js

```diff
@@ -21,11 +21,14 @@ const supportPlugin = async (pkg) => {
     ? JSON.parse(support.contents.toString())
     : support.contents;
 
+  const targets = ['all', 'lts', 'active', 'lts_active', 'supported'];
+
   if (supportData !== 'unknown') {
     // Check for LTS node target
-    const targetsLTS = supportData.versions?.find(
-      (item) => item?.target?.node === 'lts'
-    );
+    const targetsLTS = supportData.versions
+      .map((version) => version?.target?.node)
+      .some((target) => targets.indexOf(target) >= 0);
```

Comment:
You can use `includes()` instead of `indexOf()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
```suggestion
      .some((target) => targets.includes(target));
```

Reply:
Ahh, I always forget that the `includes` method exists 🤦‍♂️
@richardlau I have a feeling that this is not the first time you mention this to me 😅

Reply:
There's probably a lot of old articles/examples out there using Array.indexOf but Array.includes has been in Node.js since 6.0.0.